recipes/glue/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "lscsoft-glue" %}
+{% set version = "1.58.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7002676d2989f959bfae53cce622eb528acc82cc9d43e76a2c850e1577e1aaf4
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python {{ python }}
+    - setuptools
```

Comment:
remove setuptools here since pip contains it

Reply:
I would rather be explicit about dependencies, rather than implicit, since `lscsoft-glue` needs `setuptools` regardless of whether you use `pip` or not, but I don't feel too strongly.